flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/StormTupleTest.java

```diff
@@ -595,7 +593,7 @@ public void testGetBinaryByFieldPojoGetter() throws Exception {
 	private <T> StormTuple testGetByField(int arity, int index, T value)
 			throws Exception {
 
-		assert (index < arity);
+		Assert.assertTrue(index < arity);
 
```

Comment:
Hmm, it's part of the test, no? Doesn't really matter but in general I thought usage of assert is discouraged.